nifi-nar-bundles/nifi-grpc-bundle/nifi-grpc-processors/src/main/java/org/apache/nifi/processors/grpc/ListenGRPC.java

```diff
@@ -82,25 +83,29 @@
     // properties
     public static final PropertyDescriptor PROP_SERVICE_PORT = new PropertyDescriptor.Builder()
             .name("Local gRPC service port")
+            .displayName("Local gRPC Service Port")
             .description("The local port that the gRPC service will listen on.")
             .required(true)
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
     public static final PropertyDescriptor PROP_USE_SECURE = new PropertyDescriptor.Builder()
-            .name("Use SSL/TLS")
-            .description("Whether or not to use SSL/TLS to send the contents of the gRPC messages.")
+            .name("Use TLS")
```

Comment:
The `name` value needs to stay the same so that templates and flows which referenced the previous version will continue to work. If this value changes, existing templates and flows will not be read correctly and these values will be lost. This is why `displayName` was introduced.

Reply:
My rationale on that name change was that `ListenGRPC` hasn't appeared in a formal nifi release yet (I added it in the current baseline), so i went with the name change. However, i'm good with whichever way you decide.

Reply:
I will sign off on this because I agree with your reasoning, but I would appreciate "Apache" guidance from @joewitt on this. I know some people do run "bleeding-edge"/`master` releases (ideally not in production, but still...).

Reply:
The name change is entirely fair game since it hasn't been in a release yet.